syntax = "proto3";

package bayes.bob;

// The definition of an Advice Module. See http://go/bob:advisor-design.
message AdviceModule {
  // Unique readable ID for the module.
  string  advice_id = 4;

  // Name of the scoring model to decide whether to trigger this advice.
  string trigger_scoring_model = 1;

  // ID of the sticky action template to engage the user when they have
  // accepted the advice.
  string engage_sticky_action_template = 2;

  // Whether this module is fully implemented, otherwise it's only availble to
  // test users.
  bool is_ready_for_prod = 3;
}
